src/xunit.performance.api/CSVFile.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Xunit.Performance.Api.Table
+{
+    public sealed class CSVReader : IDisposable
```

Comment:
Can you mark this and all the other related classes for writing/reading CSVs as internal?  We shouldn't export this functionality for users.